native/SampleApps/RestAPIExplorer/RestAPIExplorer/MobileSDKAppConfig.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Forgot about the `bootconfig.plist` name.  Would you prefer I change it to that?  Picked this out of a hat. :)

Reply:
Yes please :-)